docs/node/migrations/zero-downtime-migrations/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 2
+sidebar_label: "Zero Downtime Migrations"
+id: zero-downtime-migrations
+title: Zero Downtime Migrations
+description: Zero Downtime Migrations
+slug: /node/migrations/zero-downtime-migrations
+---
+
+In the scenarios where the network cannot afford any downtime, Zero Downtime Migrations are the best solution. This method involves launching a new network and running it in parallel with the existing network during the migration period and streaming changes from the existing network to the new network to keep the new network in sync with the existing network. At the end of the migration period, the old network can be decommissioned without any disruption to the network's users or services.
```

Comment:
Similar to my comment on offline migrations, I think you can remove this page because it just repeats what was already said in the migrations intro.

Reply:
TBH: I think you can condense `migration-proposal.mdx`, `migration-phases.mdx`, and `state-replication.mdx` into one document. Tutorial on a separate document is good.